io/net/ethtool_test.py

```diff
@@ -62,6 +62,10 @@ def setUp(self):
         self.netmask = self.params.get("netmask", default="")
         self.hbond = self.params.get("hbond", default=False)
         self.peer = self.params.get("peer_ip")
+        self.tx = self.params.get("tx_channel", default='')
+        self.rx = self.params.get("rx_channel", default='')
+        self.other = self.params.get("other_channel", default='')
+        self.combined = self.params.get("combined_channel", default='')
```

Comment:
lets keep some default values initialize for above params instead of empty.. meaning test should run fine when no input values are given